sample/ml_sample_schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/EMBL-EBI-SUBS/validation-schemas/master/sample/ml_sample_schema.json",
```

Comment:
Remove `$id`, see google document for more details.

Reply:
okay done.